.github/get-latest-python-version.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import json
+import re
+import urllib.request as request
+
+
+def get_superior_version_tag(tag1: str, tag2: str) -> str:
+    tag1_split = tag1.split(".")
+    tag2_split = tag2.split(".")
+    for i in range(len(tag1_split)):
+        if int(tag1_split[i]) > int(tag2_split[i]):
+            return tag1
+        elif int(tag1_split[i]) < int(tag2_split[i]):
+            return tag2
+
+
+def get_latest_version_number() -> str:
+    url_tags = request.Request(
+        "https://hub.docker.com/v2/repositories/library/python/tags/?name=-slim&page_size=50"
+    )
+    tags_request = request.urlopen(url_tags)
+    json_tags = json.load(tags_request)["results"]
+
+    latest_tag = None
+    for tag in json_tags:
+        if re.fullmatch(r"^\d+\.\d+\.\d+-slim", tag["name"]):
+            tag_name = tag["name"].replace("-slim", "")
+            if latest_tag is None:
+                latest_tag = tag_name
+            else:
+                latest_tag = get_superior_version_tag(tag_name, latest_tag)
```

Comment:
or simply `sort` the tag list and take the first one, might be easier to read :)

Reply:
Indeed, done !